### DISCUSSION TITLE
Run ansible playbooks

### DISCUSSION THREAD

**Discussion by hialvaro** [windmill-labs/windmill#1587] [Questions] [answered]
Is it possible, or could it be possible, to trigger the run Ansible playbooks from windmill?

**Comment by rubenfiszel** [comment, marked as answer]
@hialvaro you could run it from bash if the ansible cli is installed on your worker. If there is a lot of demand for it, we could add it as a "supported language". Priority is powershell for now :)

**Reply by hialvaro** [reply]
Is there any docs on how to add a worker?

**Reply by rubenfiszel** [reply]
Workers pull jobs from the queue so they are automatically added as long as they are connected to the queue (so sharing the same DATABASE_URL from the servers are sufficient). You can take a look at the main docker-compose or our helm charts: https://github.com/windmill-labs/windmill-helm-charts to see how they are usually deployed

Here in this case, you would create a docker image that takes windmill as a base and install ansible on top of it.

**Reply by hialvaro** [reply]
Thanks!
